app.py

```diff
@@ -455,6 +458,7 @@ async def test():
     m = Model(
         request_function["test"],
         "test_model1",
+        1,
```

Comment:
why is it hardcoded to 1?

Reply:
Model() took different data and in third position, they took the version of the model. Here, since I'm building a test model for the testing, I set the version of the test model to 1.

In the test refactor issue (#59) we should probably delete this endpoint and build the test models in the test files directly.